sdk/keyvault/keyvault-certificates/package.json

```diff
@@ -66,7 +67,8 @@
     "test:node": "npm run clean && npm run build:test && npm run unit-test:node",
     "test": "npm run clean && npm run build:test && npm run unit-test",
     "unit-test:browser": "karma start --single-run",
-    "unit-test:node": "mocha --require source-map-support/register --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=- --timeout 2000000 --full-trace dist-test/index.node.js",
+    "unit-test:node": "mocha --require source-map-support/register --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=- --timeout 180000 --full-trace dist-test/index.node.js",
+    "unit-test:node:no-timeout": "mocha --require source-map-support/register --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=- --timeout 180000 --full-trace dist-test/index.node.js",
```

Comment:
should this be in .vscode/launch.json instead? Also mocha has a --no-timeouts option.

Reply:
.vscode/launch.json assumes devs are using VSCode 😅 I am in fact using it, but I'd like this not to be exclusive to VSCode. I'll use the mocha one though!

Reply:
As I remember, we prefer not to maintain an ever-growing list of npm scripts, if it is not too inconvenient to manually change. For example, we don't have npm scripts to help generate recordings either.